src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/rollup/settings/RollupSettings.kt

```diff
@@ -77,5 +77,12 @@ class RollupSettings {
             Setting.Property.NodeScope,
             Setting.Property.Dynamic
         )
+
+        val ROLLUP_DASHBOARDS: Setting<Boolean> = Setting.boolSetting(
+            "opendistro.rollup.dashboards",
```

Comment:
Do you think we'll have any other dashboard related settings in future? Do we want this to be "...dashboards.enabled" to allow using the "dashboards" namespace later on?

Reply:
Not sure if we are going to have any other rollup dashboard settings, but keeping namespace available for future sounds like a good idea just in case. Will change